timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -21,6 +21,13 @@ export interface TimeGraphChartProviders {
     rowStyleProvider?: (row: TimelineChart.TimeGraphRowModel) => TimeGraphRowStyle | undefined
 }
 
+export const keyBoardNavs: Record<string, Array<string>> = {
+    "Zoomin": ['w', 'i'],
```

Comment:
You could use constants here, as the capital letter scheme may easily get mixed up by future developers (start by capital and no camelcase). all caps constants would be clearer. Or make them all lowercase to match case scheme of keyboard and mouse events

Reply:
Good point. I have made them all lowercase.